.github/workflows/build-and-push-assets.yml

```
@@ -183,13 +207,13 @@ jobs:
       - name: Set branch environment variables
         if: ${{ github.ref_type == 'branch' }}
         run: |
-          echo "BUILT_BRANCH_NAME=${{ github.ref_name }}${{ inputs.BUILT_BRANCH_SUFFIX }}" >> $GITHUB_ENV
+          echo "BUILT_BRANCH_NAME=${{ github.ref_name }}${{ env.BUILT_BRANCH_SUFFIX }}" >> $GITHUB_ENV
```

Does it make sense to completely remove this env variable since it is only used once?
```suggestion
          echo "BUILT_BRANCH_NAME=${{ github.ref_name }}-built" >> $GITHUB_ENV
```